### PR TITLE
Add buildkite analytics test collection to react and react-native e2e tests

### PR DESCRIPTION
**Pull request by Cawllec**
There's some overlap with maze-runner options between config files and options in the pipeline, once the maze-runner mechanism for reading the config files is fixed we should be able to remove the pipeline options.

### CODE REVIEW

File: test/react-native/features/support/maze.all.cfg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+--no-tunnel
+--aws-public-ip
```
Comment (twometresteve) [inline test/react-native/features/support/maze.all.cfg:2]: These two should only be in place on Buildkite (BrowserStack will be used locally with the secure tunnel).

**Review by twometresteve — APPROVED** [review]
One small tweak to make before merging, but happy to approve now.

File: test/react-native/features/support/maze.all.cfg (hunk 1 above)
Comment (Cawllec) [reply test/react-native/features/support/maze.all.cfg:2]: True, but all the React-native tests use Buildkite AFAICT

Comment (twometresteve) [reply test/react-native/features/support/maze.all.cfg:2]: In principle engineers could be running tests on their local machine when writing new tests, even if it's awkward to build the test fixture.

**Pull request merged**